### DISCUSSION TITLE
Background rotation in auto-rotate mode

### DISCUSSION THREAD

**Discussion by suro7** [google/model-viewer#2177] [Q&A]
### Description

In auto-rotate mode model is rotating but the background image is still.
Would make sense to have option to toggle synchronous background rotation together with model, just like when you rotate model manually.

**Comment by cdata** [comment]
Hey @suro7 thanks for filing this feedback! I'm going to mark this as a duplicate of #576 since that was filed first.

The reason why auto-rotate doesn't currently cause the background-image to rotate is that auto-rotate turns the model on a turntable. It's possible to create the effect you want by orbiting the camera around the model instead. I'll change #576 to a feature now that we have a couple of requests for this.

In the mean time, you can manually create the effect you are looking for by updating the `camera-orbit` attribute in an animation callback. A very basic example would look like this:

```typescript
const updateOrbit = () => {
  const orbit = modelViewer.getCameraOrbit();
  modelViewer.cameraOrbit = `${orbit.theta + 0.01}rad ${orbit.phi}rad ${orbit.radius}m`;
  modelViewer.jumpCameraToGoal();
  requestAnimationFrame(() => updateOrbit());
};

updateOrbit();
```
